### PR TITLE
Issue 4 post api search

### LINKED ISSUE #4: Create POST api/search route

### PR DESCRIPTION
**Pull request by JuanMiguelRoman1008**

### CODE REVIEW

File: routes/api/search.js [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+                {
+                    user,
+                },
+            ],
+        });
+        await search.save();
+        res.status(200).send(search);
+    } catch (error) {
+        res.status(500).send(
+            `An error occured while trying to create your search, ${error}`
```
[the remark this answers is not in the document]
Comment (JuanMiguelRoman1008) [reply routes/api/search.js:27]: Done testing catch responses for each error

File: routes/api/search.js [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
+            game,
+            limit,
+            party: [
+                {
+                    user,
+                },
+            ],
+        });
+        await search.save();
+        res.status(200).send(search);
```
[the remark this answers is not in the document]
Comment (JuanMiguelRoman1008) [reply routes/api/search.js:24]: Addressed in 010a54c46afc08483945cae9a84cf2d784507642

[the remark this answers is not in the document]
Comment (JuanMiguelRoman1008) [reply routes/api/search.js:24]: I think that will be okay, but if that will add security risks, we can refactor the code

File: routes/api/search.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jonfelixrico) [reply routes/api/search.js:27]: What was the output like in the response?

File: routes/api/search.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jonfelixrico) [reply routes/api/search.js:24]: Alright. We can just do this later. We're still in the pre-alpha stage anyway.

File: routes/api/search.js [hunk 3]
```diff
@@ -1,12 +1,32 @@
...
+const Search = require("../../models/Search");
 const router = express.Router();
 
 // @route  POST api/search
 // @desc   Add Looking For request
 // @access Private
 
-router.post("/", (req, res) => {
-    res.send("Add an LF request");
+router.post("/", async (req, res) => {
```
[the remark this answers is not in the document]
Comment (JuanMiguelRoman1008) [reply routes/api/search.js:9]: Addressed in cc495a658c1d651386caca18d9954f1349dce935

File: routes/api/search.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jonfelixrico) [reply routes/api/search.js:27]: Resolving, will just do a last pass on this on the staging branch.

**Pull request merged**